package.json

```diff
@@ -11,7 +11,10 @@
   ],
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha --timeout 30000 test/*test.js"
+    "lint": "eslint .",   
+    "test": "./node_modules/.bin/mocha --timeout 30000 test/*test.js",
```

Comment:
don't need the ./node_modules stuff

`"test": "mocha --timeout 30000 test/*.test.js`